app/src/main/java/swati4star/createpdf/activity/MainActivity.java

```diff
@@ -130,6 +132,13 @@ protected void onCreate(Bundle savedInstanceState) {
         openWelcomeActivity();
     }
 
+    @Override
+    protected void onDestroy() {
+        super.onDestroy();
+        // clear the temp directory on app exit
```

Comment:
You can remove this comment. The name of the method makes the intentions pretty clear.

Reply:
sure